src/test/resources/features/convertImage/image.feature

```diff
@@ -0,0 +1,283 @@
+Feature: Image converter
+
+  In order to convert a image file
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set authentication headers
+
+  @functional
+  Scenario: Verify that a image file is converted to another image file with a different extension
```

Comment:
Please consider improving this scenario title to something like: `Verify that an image file is converted to PNG image format`